vuepress/.vuepress/config.js

```diff
@@ -5,7 +5,12 @@ module.exports = {
       lang: 'en-US',
       title: 'Vue I18n',
       description: 'Vue I18n is internationalization plugin for Vue.js'
-    }
+    },
+    '/zh/': {
+      lang: 'zh-CN',
+      title: 'Vue I18n',
+      description: 'Vue I18n是Vue.js的国际化插件'
```

Comment:
https://github.com/vuejs/cn.vuejs.org/wiki
这里的 wiki 是一些之前团队对中文文档的约定，可以做个参考，比如这里对空格的处理：

```suggestion
      description: 'Vue I18n 是 Vue.js 的国际化插件'
```

Reply:
感谢您的校对，今晚我会再参考着这个规范，再次对译文进行一次重新的排版